### PR TITLE
Molr project

### PR DESCRIPTION
**Pull request by andreacalia**
This pull request adds the Molr project proposal for GSOC 2019

### CODE REVIEW

**Comment by JavierCVilla** [comment]
As a personal comment, both the descriptions and tasks look good to me. However, I think it may be difficult to digest for a student without background on a similar system due to the terminology (Moles, missions, supermole). Maybe an additional paragraph on the description with a more general view of the expected results or a brief example would increase chances of getting more students interested on the project.

Said that, this is not a request to have this proposal merged.

**Comment by andreacalia** [comment]
> @andreacalia Thank you for submitting this proposal.
> 
> Could you please add a logo for the Molr project? You can add it to [`images`](https://github.com/HSF/hsf.github.io/tree/master/images).

The project is still WIP but we could have a temporary logo, thanks :)

**Comment by andreacalia** [comment]
> As a personal comment, both the descriptions and tasks look good to me. However, I think it may be difficult to digest for a student without background on a similar system due to the terminology (Moles, missions, supermole). Maybe an additional paragraph on the description with a more general view of the expected results or a brief example would increase chances of getting more students interested on the project.
> 
> Said that, this is not a request to have this proposal merged.

Thanks for the comments, there is a full explanation in the project's website but you're right that we should improve it in the description of the proposal. We'll work on it :)

File: _gsocprojects/2019/project_MOLR.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+project: Molr
+layout: default
+logo: molr-logo.png
+description: |
+  A Modular Distributed Execution and Debugging Framework.
+
+  Molr provides a unified way to interact (locally or remotely) with executable code in order to run it and/or debug it.
+  It is possible to provide input to it and retrieve output, return values and exceptions from this.
+  The reference implementation (in the molr repository) is implemented in java. 
+  However, remote interaction is performed through a well defined REST API. This makes it very easy to implement 'plugins' ([Moles](#moles)) in any programming language.
```
Comment (agheata) [inline _gsocprojects/2019/project_MOLR.md]: The link #moles is undefined and generates an error upon merge. Can you please update with a valid external link?

Comment (andreacalia) [reply _gsocprojects/2019/project_MOLR.md]: Fixed, thanks

**Pull request merged**